spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/ThrowableCollector.kt

```diff
@@ -25,10 +25,10 @@ class ThrowableCollector {
 
     fun isEmpty() = throwable == null
 
-
     fun assertEmpty() {
-        if (throwable != null) {
-            throw throwable!!
+        throwable?.let {
+            if (it !is AssertionError) it.printStackTrace()
```

Comment:
@artem-zinnatullin but there's another place where the stacktrace is been printed?